contracts/src/v0.8/functions/dev/1_0_0/RouterBase.sol

```diff
@@ -106,20 +96,9 @@ abstract contract RouterBase is IRouterBase, Pausable, ITypeAndVersion, Confirme
     // NOTE: this cannot be later modified
```

Comment:
If this cannot be modified why isn't this immutable?

Reply:
Also made it private